discord/state.py

```diff
@@ -281,7 +281,10 @@ def parse_guild_member_remove(self, data):
     def parse_guild_member_update(self, data):
         server = self._get_server(data.get('guild_id'))
         user_id = data['user']['id']
-        member = server.get_member(user_id)
+        try:
+            member = server.get_member(user_id)
+        except AttributeError:
```

Comment:
I throse it for me when I use client.accept_invite with the url or code someone messages it when it is not supposed to crashign the bot. When I did this it works fine.